### DISCUSSION TITLE
SoloV2 problem

### DISCUSSION THREAD

**Discussion by ptman** [solokeys/solo2#125] [Q&A]
Hi!

I have solov1 and solov2. I'm developing a webauthn app with resident keys/discoverable credentials/passkeys. It works with solov1 and chrome built-in virtual token, but fails with solov2. I'm getting:

Uncaught (in promise) DOMException: The operation either timed out or was not allowed. See: https://www.w3.org/TR/webauthn-2/#sctn-privacy-considerations-client.

Is this a known bug? I tried upgrading, but it seems I'm running the latest code.

**Comment by nickray** [comment]
Do things work on https://webauthn.io/?

**Comment by ptman** [comment]
Does webauthn.io have a demo of discoverable credentials? I didn't find one. But I'm using the same https://pkg.go.dev/github.com/duo-labs/webauthn library on the server.

**Comment by ptman** [comment]
The fresh firmware update didn't help (and the changelog didn't indicate that it should help)

**Comment by nyabinary** [comment]
cc: @nickray
